mendezona/tradingview-orders
# 010 | Add alpaca integration

Added more to Alpaca Markets integration. Added helper functions to do the following functions:

- Get balance of a single asset
- Get latest quote for an asset
- Check if asset is fractionable
- Submit Market Order for custom amount
- Submit Market Order for custom % of portfolio worth
- Check if last order for an asset was a sell
- Save taxable profits to DynamoDB table
- Subtract taxable profits for Market Order portfolio calculation
- Sell all holdings of an asset
